src/chart/bar/BarChart.tsx

```diff
@@ -156,7 +185,11 @@ const NeoBarChart = (props: ChartProps) => {
     }
 
     return (
-      <g transform={`translate(${bar.x},${bar.y})`}>
+      <g transform={`translate(${bar.x},${bar.y})`} 
+      // onClick event to trigger event to pass value with report action
+      onClick={(event) => onClick(bar.data, event)}
+      style={{ cursor: 'pointer' }}
```

Comment:
this seems to have not changed anything, my cursor is the same as before

Reply:
pointer only works if the position label != 'off'. Changed it now so it always works.